### PR TITLE
Master

### PR DESCRIPTION
**Pull request by garykac**
Add support for 'uievents' W3C group

### CODE REVIEW

File: bikeshed/config.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 groupsInW3C = frozenset(["csswg", "dap", "fxtf", "geolocation", "houdini",
                          "html","ricg", "svg", "texttracks", "web-bluetooth-cg",
-                         "webappsec", "webspecs", "whatwg"])
```
Comment (tabatkins) [inline bikeshed/config.py]: Why did you remove webappsec?

Comment (garykac) [inline bikeshed/config.py]: That was not intentional. It should only be adding support for uievents.
